ohno.md

```diff
@@ -108,12 +108,16 @@
 
 ... someone converts the country of Canada to a bus stop.
 
+... someone converts a bus stop to a country.
+
 ... someone adds `public_transport:version=3` to a bus route relation.
 
 ... someone maps multiple tennis courts as one.
 
 ... you cannot delete any OSM elements for one month.
 
+... you cannot add any OSM elements for one month.
+
```

Comment:
I think duplicating and adding opposite "ohno"s isn't too helpful, it's just going to lower the chances of the others being selected and just adds unceccecary ones to the list.
(same applies to the ones further down)